File: 231115/숫자들의 배수/multiple-of-numbers.java
import java.util.*;
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());
        int num = Integer.parseInt(st.nextToken());
        int result = num;
        int count = 0;
        StringBuilder sb = new StringBuilder();
        while (count < 2) {
            if (result % 5 == 0) {
                count++;
            }
            sb.append(result).append(' ');
            result += num;
        }
        System.out.println(sb);
    }
}

File: 231115/숫자들의 배수/multiple-of-numbers.js
const fs = require('fs');
const num = Number.parseInt(fs.readFileSync(0).toString().trim());

let arr = [];
arr.push(num);
for (let i = 1; i < 10; i++) {
    arr[i] = arr[i-1] + num;
}

let count = 0;
let result = [];
for (let i = 0; i < 10; i++) {
    if (arr[i] % 5 === 0) {
        count++;
    }
    result.push(arr[i]);
}

console.log(result.join(' '));